src/components/bone-visibility.js

```diff
@@ -16,6 +16,8 @@ AFRAME.registerComponent("bone-visibility", {
       }
 
       this.lastVisible = visible;
+      this.el.object3D.updateMatrices(true, true);
+      this.el.object3D.updateMatrixWorld(true, true);
```

Comment:
answered in discord but since its not visible, we won't update the matrices, which matters because then it will be mis-skinned. (this took me forever to actually fix given the hairy nature of getting the matrix update routines both correct and fast)

the visibilty changes happen rarely enough I think it'd be more confusing than its worth to special case it for the transition -- in general this just reads "ensure all the matrices are always correct coming out of this function if visibility changes"